geoutils/raster/distributed_computing/multiproc.py

```diff
@@ -189,24 +195,34 @@ def map_overlap_multiproc_save(
 
     # get first tile to retrieve dtype and nodata
     result_tile0, _ = config.cluster.get_res(tasks[0])
+    file_metadata = {
+        "driver": "GTIFF",
```

Comment:
The fact that it's constantly defined here makes me think the "driver" could become a parameter of the `MultiprocConfig` eventually?